docs/examples/asyncagi/example.php

```diff
@@ -48,8 +47,7 @@
 ////////////////////////////////////////////////////////////////////////////////
 // Mandatory stuff to bootstrap.
 ////////////////////////////////////////////////////////////////////////////////
-require_once 'PAMI/Autoloader/Autoloader.php'; // Include PAMI autoloader.
-\PAMI\Autoloader\Autoloader::register(); // Call autoloader register for PAMI autoloader.
+require_once 'autoload.php'; // Include Composer autoloader.
```

Comment:
mm relative paths in `require` and `include` are rather ugly. It's a bit nicer to have a specific include_path defined, and then require without the path